docs/debugger/intellij.rst

```diff
@@ -0,0 +1,105 @@
+Debug Open edX with IntelliJ IDEA and Add a Python Remote SDK
```

Comment:
For clarity, this is specific to edx-platform, right? I think we should mention that explicitly in the heading.